lib/app.dart

```diff
@@ -1,15 +1,29 @@
+import 'package:InstiComplaints/login.dart';
+import 'package:InstiComplaints/register.dart';
+
 import 'package:flutter/material.dart';
 import 'st_profile.dart';
-
+import 'ad_profile.dart';
 import 'filed.dart';
 import 'resolved.dart';
+import 'Compose.dart';
+import 'notifications.dart';
+import 'navigation.dart';
+import 'login.dart';
```

Comment:
The file cannot be compiled due to multiple imported classes with the same name. Delete this line!